Help_Desk_process.md

```diff
@@ -1,44 +1,44 @@
 # Process for Help Desk Requests from External Users
 
+> **SUMMARY**: If a Help Desk issue is assigned to you, then within two business days you are expected to add a comment to the issue that answers the questions, "Is this something that I can address? If so, when?"
+
 When a new issue is received from an external user, it automatically creates a JIRA ticket.
-Help Desk managers send a generic response (within one day) to the user notifying them that we will get them an answer soon
-Unless the Help Desk managers can answer the question themselves, they assign the issue to the relevant box lead.
-The box lead assigns external issues within two days to someone who can address the issue.
+Help Desk managers send a generic response (within one day) to the user notifying them that we will get them an answer soon.
+Unless the Help Desk managers can answer the question themselves, they assign the issue to the person they think is most likely to be able to address it.
 
-Within three business days, the assignee looks at the assigned issue and triages it, entering a comment in the ticket to indicate:
+**Within two business days, the assignee needs to look at the assigned issue and enter a comment in the ticket to indicate whether they are the right assignee**. If they are not, then they should indicate in a comment that they can't address it, and they should reassign to the right person if they know who that is, or say that they don't know who it should be assigned to.
```

Comment:
I would rephrase a bit for readability as

Within two business days, the assignee needs to comment to either:
* acknowledge that they will handle the issue (even if not immediately), or
* indicate that they are not the right person to do so.  In this case, They should reassign to the right person if they know who that is, or request that it be reassign if they don't.